changelogs/CHANGELOG-1.3.0.md

```diff
@@ -0,0 +1,4 @@
+## All changes
+
+This release contains no user facing changes but includes fixes for Change the base image to distroless.
+CVE-2021-3121 and CVE-2021-3580
```

Comment:
Remove this line

Reply:
I'm curious about it that the PR for cve was just merged a few hours ago